src/layer/VideoOverlay.js

```diff
@@ -71,6 +75,7 @@ export var VideoOverlay = ImageOverlay.extend({
 		if (!this.options.keepAspectRatio && vid.style.hasOwnProperty('objectFit')) { vid.style['objectFit'] = 'fill'; }
 		vid.autoplay = !!this.options.autoplay;
 		vid.loop = !!this.options.loop;
+		vid.muted = this.options.muted;

```

Comment:
Should be coerced into a `Boolean` with `!!`, as with the autoplay and loop options. Other than that, it's a good addition :-)

Reply:
@IvanSanchez , I fixed it.